ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v3/fileinput/README.md

```diff
@@ -53,10 +55,10 @@ The following properties are written to JCR for this Form File component and are
 19. `./dragDropText` - defines the file input drag and drop area title 
```

Comment:
This needs to be changed to `fd:dragDropText`

Reply:
I think its fine to let the property name remain as `dragDropText`. We have changed it in exporter to fd:dragDropText which IMO is sufficient for consistency.

Reply:
Yes, but since we were already changing this behaviour, its better to align with the new practise of naming OOTB property names